modules/installation-configuration-parameters.adoc

```diff
@@ -520,6 +520,10 @@ This property is deprecated. To use a flavor as the default for all machine pool
 On clusters that use Kuryr, {rh-openstack} Octavia does not support availability zones. Load balancers and, if you are using the Amphora provider driver, {product-title} services that rely on Amphora VMs, are not created according to the value of this property.
 |A list of strings. For example, `["zone-1", "zone-2"]`.
 
+|`controlPlane.platform.openstack.rootVolume.zones`
```

Comment:
Shouldn't it be `compute.platform.openstack.rootVolume.zones`?

Reply:
🙊 🙇 Thanks! @mandre